### ISSUE TITLE
[AppBar] Voice over order in when using MDCAppBar

### ISSUE THREAD

**Issue by randallli** [material-components/material-components-ios#2897]
### Overview

When voice over is on the top left navigation item should be the first thing read when entering a view controller with an App Bar. This is not the case as it reads items within the scroll view.

### Reproduction steps
Turn on voice over in settings.
Navigate to the App Bar component
Open the "UITableView with section headers" example

#### unexpected behavoir
notice that the voice over has selected the scroll categories

#### expected behavior
It should select the top left navigation item as the focused item as is the default on the iOS platform.

**Comment by AlimoRabbani** [comment]
@jverkoey Do you have any thoughts on how this can be fixed? I can contribute, but I don't have any possible solutions in mind.

**Comment by AlimoRabbani** [comment]
I think the initial selection is not the only issue here. The ordering of a11y elements is also wrong.

**Comment by AlimoRabbani** [comment]
The issue in AppBarSectionHeadersExample is that the MDCAppBar is added as a subview of the table view, which is the root view of the UITableViewController. iOS applies transforms to subviews of the table view to support scrolling and it seems to break hierarchy flattening for a11y.

Adding a parent UIView and making the MDCAppBar and UITableView siblings fixes the problem in this case.

**Comment by jverkoey** [comment]
Duplicate of https://github.com/material-components/material-components-ios/issues/3660.
